cmd/tetragon/main.go

```diff
@@ -457,6 +461,10 @@ func execute() error {
 	// Allow to disable kprobe multi interface
 	flags.Bool(keyDisableKprobeMulti, false, "Allow to disable kprobe multi interface")
 
+	// Allow to specify perf ring buffer size
+	flags.Int(keyRBSizeTotal, 0, "Set perf total ring buffer size (default 65k per cpu)")
+	flags.Int(keyRBSize, 0, "Set perf cpu ring buffer size (default 65k)")
```

Comment:
Based on my understanding this is not `(default 65k)` but `(default 65k * NUM_CPUS)`. Right?

Reply:
argh yea.. copy/paste issue :-\ will fix that, thanks